### ISSUE TITLE
Android borderLeftWidth and borderLeftColor broken 

### ISSUE THREAD

**Issue by Jpoliachik** [facebook/react-native#5411]
I am attempting to use a css trick to produce triangles using the code: 
```
'use strict';

var React = require('react-native');
var {
  AppRegistry,
  StyleSheet,
  Text,
  View,
} = React;

var SampleApp = React.createClass({
  render: function() {
    return (
      <View style={styles.container}>
        <View style={styles.triangle}/>
      </View>
    );
  }
});

var styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#F5FCFF',
  },
  triangle: {
     width: 0,
     height: 0,
     backgroundColor: 'transparent',
     borderStyle: 'solid',
     overflow: 'hidden',
     borderTopWidth: 50,
     borderRightWidth: 0,
     borderBottomWidth: 50,
     borderLeftWidth: 100,
     borderTopColor: 'transparent',
     borderRightColor: 'transparent',
     borderBottomColor: 'transparent',
     borderLeftColor: 'red',
   },
});

AppRegistry.registerComponent('SampleApp', () => SampleApp);
```

In iOS this produces: 
![screen shot 2016-01-19 at 10 41 26 am](https://cloud.githubusercontent.com/assets/5148640/12423394/3ffcb970-be99-11e5-8527-4c8030aec41e.png)

But in Android, it only shows a black square:
![screen shot 2016-01-19 at 10 40 21 am](https://cloud.githubusercontent.com/assets/5148640/12423398/457e8fcc-be99-11e5-9bad-517d68e8dccf.png)

Does Android support using these styles?
It doesn't seem to like `borderLeftWidth`, `borderLeftColor`, etc. 

I don't see anything related in the documented Known Issues. 

Edit: The screenshots are from RNPlay running react-native 0.13.0-rc.

**Comment by dmmiller** [comment]
Just had a look at the code and these are not currently supported on Android. Have filed a task internally (t9789502) to look further at supporting it at some point.

**Comment by dmmiller** [comment]
Anyone up for doing a PR for this?

**Comment by trave7er** [comment]
anyone doing this already?

**Comment by mobileDevNativeCross** [comment]
hey!!!
anyone doing this already?

**Comment by dmmiller** [comment]
Not that I know of.  Feel free to jump on it :)

**Comment by trave7er** [comment]
@mobileDevNativeCross , @dmmiller , I did some digging last weekend, it seems we can do it [here at Line 348] (https://github.com/facebook/react-native/blob/4937a4c5cd2494a875019bfeb691026b0abfe58e/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java#L348). 

The reasoning would be that if we are drawing rectangles, we never gonna have a triangle any where.

**However**, if we use `canvas.drawPath`, instead of `canvas.drawRect`, we will be changing the default behavior of all the borders with different colors. Would love to have your options on this before jumping in.

**Comment by trave7er** [comment]
hi guys, if no one has started working on this issue by the end of today, I'll create a PR this weekend. @dmmiller , @mobileDevNativeCross

**Comment by trave7er** [comment]
Please take a look at PR #5911 @mobileDevNativeCross @dmmiller 

Thanks,

Huang

**Comment by trave7er** [comment]
I think this issue can be closed by #5911 , right?

**Comment by christopherdro** [comment]
Appears to fixed via https://github.com/facebook/react-native/commit/a6a4389bf4a581de5adfe0e5dda4aa4f70438dfc

Feel free to re-open if problem persists.
